lib/resty/redis.lua

```diff
@@ -115,6 +115,10 @@ function _M.connect(self, ...)
         ok, err = sock:connect(host, port, connect_opts)
     end
 
+    if not ok then
+        return nil, "failed to connect: " .. (err or "")
```

Comment:
Thank you for adding this check.
Travis CI is failed now. I check the travis build and see that is message error.
```
got: 'failed to connect: failed to connect: connection refused
expected: 'failed to connect: connection refused
```
Maybe just change the error message to `return nil, "(err or "")` or change the error message at https://github.com/openresty/lua-resty-redis/blob/734abe1d4ecc02d0efb2751bfd9f9a9dddc69719/t/sanity.t#L845

Reply:
updated this

Reply:
still got the same error.

Reply:
my bad. I pushed empty commit earlier, tests are passing now!